bot/worker.py

```diff
@@ -0,0 +1,184 @@
+"""Main module of the Dota 2 subreddit Responses Bot.
+
+The main body of the script is running in this file. The comments are loaded from the subreddit
+and the script checks if the thing is a response from Dota 2. If it is, a proper thing is
+prepared. The thing is posted as a reply to the original post on Reddit.
+
+Proper logging is provided - saved to 2 files as standard output and errors.
+"""
+import string
+
+from praw.models import Comment
+
+import config
+from bot import account
+from util.caching import get_cache_api
+from util.database.database import db_api
+from util.logger import logger
+
+__author__ = 'Jonarzz'
+__maintainer__ = 'MePsyDuck'
+
+cache_api = get_cache_api()
+
+
+def work():
+    """Main method executing the script.
+
+    It connects to an account, loads dictionaries from proper files (declared in config file).
+    Afterwards it executes process_comments method with proper arguments passed.
+    """
+
+    reddit = account.get_account()
+    logger.info('Connected to Reddit account : ' + config.USERNAME)
+
+    comment_stream = reddit.subreddit(config.SUBREDDIT).stream.comments(pause_after=-1)
+    submission_stream = reddit.subreddit(config.SUBREDDIT).stream.submissions(pause_after=-1)
+    while True:
+        for comment in comment_stream:
+            if comment is None:
+                break
+            process_thing(reddit, comment)
```

Comment:
I mainly went with it just because of the reddit naming. To be honest I personally found it odd too, but just decided to roll with it anyways. I think just 'respondable' would fit our case. I'll go ahead and make the changes.

Reply:
Refactored in many places from `Thing` to `replyable`. Not the best solution, but seems appropriate to me. Didn't change it in cache as it didn't seem necesary (no context there).
Fixed in commit 41e1416fd2c7051547d1ef4807ac17741893a239